eogrow/core/area/base.py

```diff
@@ -121,20 +138,12 @@ def get_grid_cache_filename(self) -> str:
 
     def get_patch_list(self) -> PatchList:
         """Returns a list of eopatch names and appropriate BBoxes."""
-        relevant_patches = None
-        if self.config.patch_list is not None:
-            folder_path = self.storage.get_folder(self.config.patch_list.input_folder_key)
-            patch_list_path = fs.path.join(folder_path, self.config.patch_list.filename)
-            relevant_patches = set(load_eopatch_names(self.storage.filesystem, patch_list_path))
 
         named_bboxes = []
         for crs, grid in self.get_grid().items():
```

Comment:
I remember us saying something about the possibility of having a flag in the `get_grid()` function, to still enable the user/developer getting the original grid/patches, regardless of the patch list being present or not.

something along the lines of:

```python
def get_grid(self, filtered: bool = True) -> Dict[CRS, gpd.GeoDataFrame]:
```

or the default could even be false, to retain the same behavior as in the current branch

Reply:
i actually had that in mind, all the way up to the point where i got to doing it, and then promptly forgot.

added now